src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/validation/MetafileJsonValidatorTest.java

```diff
@@ -42,6 +42,26 @@ public void should_successfully_map_json_file_to_entities() throws IOException {
             );
     }
 
+    @Test
+    public void should_successfully_map_probate_json_file_to_entities() throws IOException {
+        InputEnvelope envelope = getEnvelope("/metafiles/valid/probate-from-spec.json");
+
+        assertThat(envelope.nonScannableItems).hasSize(1);
+        assertThat(envelope.scannableItems).hasSize(3);
+        assertThat(envelope.payments).hasSize(1);
+        assertThat(envelope.payments.get(0).amount).isEqualTo(new BigDecimal("100.00"));
+        assertThat(envelope.caseNumber).isEqualTo("1111222233334459");
+        assertThat(envelope.classification).isEqualTo(Classification.SUPPLEMENTARY_EVIDENCE);
+        assertThat(envelope.scannableItems)
+            .extracting("documentType")
+            .containsExactlyInAnyOrder(
+                InputDocumentType.CHERISHED,
+                InputDocumentType.OTHER,
+                InputDocumentType.WILL
```

Comment:
This document type should be added to the previous test and this test as such should be removed because it repeats a lot of assertions that are not related with jurisdiction.